src/key.rs

```diff
@@ -364,7 +362,7 @@ impl RsaPrivateKey {
 
     /// Decrypt the given message.
     pub fn decrypt<P: PaddingScheme>(&self, padding: P, ciphertext: &[u8]) -> Result<Vec<u8>> {
-        padding.decrypt(Option::<&mut DummyRng>::None, self, ciphertext)
+        padding.decrypt(None, self, ciphertext)
```

Comment:
A fringe benefit here is getting rid of some `DummyRng` usages. With a little more work it might be possible to completely get rid of it.